modules/standard/GMP.chpl

```diff
@@ -17,25 +17,157 @@
  * limitations under the License.
  */
 
-/* GMP Module 
+/*
+
+This module supports integration with the GMP library (the GNU Multiple
```

Comment:
At the moment, the generated Module Index page uses this line as its summary for the module.  If that isn't what you want, you could add a summary line at the top.